packages/backend/apps/multitenancy/tests/factories.py

```diff
@@ -0,0 +1,23 @@
+import factory
```

Comment:
I might be getting too far ahead with this but maybe we should automatically create a tenant + membership when you create a user with UserFactory?